drive/modules/listener/listener.bal

```diff
@@ -27,12 +27,34 @@ import ballerina/time;
 # + domainVerificationFileContent - File content of HTML file used in domain verification.
 # + callbackURL - Callback URL registered.  
 # + clientConfiguration - Drive client connecter configuration.
+# + retryConfiguration - Retry configuration.
+@display{label: "Listener Config"}
 public type ListenerConfiguration record {
+    @display{label: "Port"}
     int port;
+    @display{label: "Callback URL"}
     string callbackURL;
+    @display{label: "Domain Verification File Content"}
     string domainVerificationFileContent;
     Configuration clientConfiguration;
+    @display{label: "Specific Folder ID"}
     string? specificFolderOrFileId = ();
+    RetryConfiguration retryConfiguration?;
+};
+
+# Retry configuration
+#
+# + retryCount - Maximum number of reties allowed.
+# + retryInterval - Interval time between retries.  
+# + initialWaitTime - Initial wait time for domain verification.  
+@display{label: "Retry Config"}
+public type RetryConfiguration record {
+    @display{label: "Retry Count (s)"}
+    int retryCount?;
+    @display{label: "Retry Interval (s)"}
+    int retryInterval?;
+    @display{label: "Initial Wait Time (s)"}
+    int initialWaitTime?;
```

Comment:
`domainVerificationDelay` ?

Reply:
Yes. I will change accordingly.
Btw how about the display annotation `Initial Wait Time`
if we change it as `Domain Verification Delay`
That also seems good right. So the user is aware of the cause for the exact delay.

Reply:
Changed the record as follows
```
# Retry configuration
#
# + retryCount - Maximum number of reties allowed to renew listener channel. (default : 20 seconds)
# + retryInterval - Time between retries to renew listener channel in seconds. (default: 100 seconds)  
# + domainVerificationDelay - Initial wait time for domain verification check in seconds (default: 300 seconds)  
@display{label: "Retry Config"}
public type ChannelRenewalConfig record {
    @display{label: "Retry Count"}
    int retryCount?;
    @display{label: "Retry Interval"}
    int retryInterval?;
    @display{label: "Domain Verification Delay"}
    int domainVerificationDelay?;
}; ```